R/pkgs/importation/DESCRIPTION

```diff
@@ -0,0 +1,13 @@
+Package: importation
```

Comment:
Assuming this package is where @shauntruelove's code is intended to go?  If that's the case, then might be better to leave it alone since that code is already in a separate repository.

Reply:
yeah, I created it for the code in the `R` directory that was related to airport importations-- are you saying that code already exists elsewhere and can simply be removed? If so, I'll just take this stuff out for now.